jsonschema-core.xml

```diff
@@ -455,6 +455,68 @@
 
         </section>
 
+        <section title="Principles">
+            <t>
+                JSON Schema is built on several principles, which both explain various design
+                choices, and provide guidance on how to extend the specification effectively.
+            </t>
+            <!-- Links will be added to relevant spec parts after the sections are reorganized
+                 to create the new schema loading section -->
+            <t>
+                <list>
+                    <t>
+                        All syntactically legal JSON documents with well-defined behavior are fully
+                        supported as instance data.  Notably, JSON objects with duplicate property
+                        names are syntactically legal but not well-defined.
+                    </t>
+                    <t>
+                        JSON Schema is defined over a data model, not JSON text.  Therefore
+                        JSON Schema can be used with any data format that can be reliably parsed
+                        into the data model.
+                    </t>
+                    <t>
+                        Locations in schemas and instances are always identified by URIs,
+                        JSON Pointers, or Relative JSON Pointers.  JSON Schema treats URIs as
```

Comment:
Sorry, I only just noticed this now. Are relative json pointers now fair game to be used inside $refs? I think this is the first time that's been mentioned explicitly in the spec.  (The metaschema itself just uses 'format: uri-reference' for the $ref syntax, and there are no examples of relative JSON pointers used.)  It would be good to clarify this explicitly (probably in a separate patch).

Reply:
@karenetheridge ignore this PR, it's way outdated.  I should really just take it down, I may or may not revisit it, or this info may go on the web site, or something id.